docs/user/icl/local-vc-authentication.inc

```diff
@@ -1,15 +1,63 @@
+| Method               | Advantages                                         | Disadvantages                                                 |
++======================+====================================================+===============================================================+
+| Password             | - easy to use                                      | - slow with external authentication                           |
+|                      |                                                    | - potentially very insecure (when user has weak password)     |
+|                      |                                                    | - not available to all universities                           |
++----------------------+----------------------------------------------------+---------------------------------------------------------------+
+| User Token           | - easy to use, simple setup and revocation         | - dangerous if it gets leaked                                 |
+|                      | - useful for repository access via API             |                                                               |
+|                      | - very secure                                      |                                                               |
++----------------------+----------------------------------------------------+---------------------------------------------------------------+
+| Participation Token  | - easy to use, no setup needed                     | - only usable for one repository                              |
+|                      | - very limited access scope                        |                                                               |
+|                      | - very secure                                      |                                                               |
++----------------------+----------------------------------------------------+---------------------------------------------------------------+
+| SSH                  | - very secure                                      | - slower than HTTPS                                           |
+|                      | - industry standard for use with git               | - can be difficult to use                                     |
+|                      | - easy to use after correct set up                 | - can be overwhelming for new users                           |
++----------------------+----------------------------------------------------+---------------------------------------------------------------+
+
+The participation token is the preferred authentication method for students due to its simplicity, security, and lack of setup requirements.
+While SSH offers strong security, its complexity, setup challenges, and higher computational overhead make it less ideal.
+Password usage is discouraged for security reasons.
```

Comment:
That's a bit too strong, we still use it as default. I would rather wright something. Admins should disable password authentication if possible, or implement strong password rules.